mobile/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm very new on setup the package config, let me know if there is anything worth noticing